packages/vaadin-overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -80,12 +80,28 @@ export const PositionMixin = (superClass) =>
     }
 
     __overlayOpenedChanged(opened) {
-      const func = opened ? 'addEventListener' : 'removeEventListener';
-      window[func]('scroll', this.__boundUpdatePosition);
-      window[func]('resize', this.__boundUpdatePosition);
+      if (opened) {
+        window.addEventListener('scroll', this.__boundUpdatePosition);
+        window.addEventListener('resize', this.__boundUpdatePosition);
+      } else {
```

Comment:
Maybe `else if (wasOpened)` would be better?

Reply:
`removeEventListener` is a no-op in case the `opened` changes from `undefined` to `false` on init. Since this isn't a "simple observer" (but defined in `observers` array), tracking `wasOpened` would require extra logic which feels a bit overkill just to cover the initial value case.

Reply:
Thanks, makes sense.